about.css

```diff
@@ -0,0 +1,430 @@
+}
+
+.coachella-logo {
+  margin-top: 2.4rem;
+}
+
+.coachella-logo img {
+  width: 100%;
+  border: 1px solid #d6d6d6;
+  padding: 3.2rem 6.4rem;
+}
+
+.primary-header {
+  font-weight: 400;
+  font-size: 1.8rem;
+  line-height: 3.8rem;
+  color: #ec5242;
+}
+
+.headline-info {
+  text-transform: uppercase;
+  font-size: 2.8rem;
+  color: #ec5242;
+  font-weight: 700;
+  margin-bottom: 1.2rem;
+  text-align: center;
+}
+
+.concert-info {
+  font-size: 1.2rem;
+  font-weight: 400;
+  line-height: 1.6;
+  letter-spacing: 0.75px;
+  border: 2px solid #fff;
+  padding: 1.2rem;
+  margin-bottom: 2.4rem;
+}
+
+.music-icon {
+  height: 3.2rem;
+  width: 3.2rem;
+  color: #ec5242;
+}
+
+.main-nav {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 3.2rem;
+}
+
+.contact-info {
+  font-size: 1.4rem;
+  line-height: 1.4;
+  font-weight: 400;
+  text-align: center;
+}
+
+.contact-info span {
+  text-decoration: underline;
+  font-weight: 700;
+}
+
+.showcase-section {
+  padding: 4.8rem;
+  border-top: 1px solid #d6d6d6;
+}
+
+.showcase-thumbnail {
+  position: relative;
+  height: 30vh;
+  margin: 0 2rem 2rem 2rem;
+  cursor: pointer;
+}
+
+.showcase-thumbnail img {
+  width: 100%;
+  height: 100%;
+}
+
+.showcase-title {
+  font-size: 2.4rem;
+  font-weight: 400;
+  margin-bottom: 1.8rem;
+  text-align: center;
+  position: relative;
+}
+
+.showcase-title::before {
+  content: "";
+  display: block;
+  position: absolute;
+  bottom: -35%;
+  width: 5%;
+  left: 47%;
+  border-bottom: 1px solid #ec5242;
+}
+
+.showcase-info {
+  font-size: 1.4rem;
+  line-height: 1.2;
+  text-align: center;
+  margin-bottom: 1.2rem;
+}
+
+.showcase-thumbnail .description {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  color: #fff;
+  background-color: rgba(224, 104, 104, 0.6);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 0.5rem;
+  font-family: "COCOGOOSE", sans-serif;
+  transition: all ease-in-out 150ms;
+}
+
+.showcase-thumbnail .description:hover {
+  background-color: rgba(224, 104, 104, 0.2);
+}
+
+.section-headline {
+  border-bottom: 1px solid #d6d6d6;
+}
+
+.partners-heading {
+  color: #fff;
+  font-size: 1.8rem;
+  text-align: center;
+  margin-bottom: 2.4rem;
+}
+
+.partners-section {
+  background-color: rgb(39, 42, 50, 0.85);
+  padding: 4.8rem 0 3.2rem 0;
+}
+
+.partners-logo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 2.4rem;
+  list-style: none;
+  flex-wrap: wrap;
+  margin: 0 2rem;
+  column-gap: 1.5rem;
+}
+
+.partners-logo img {
+  width: 5.2rem;
+}
+
+.section-footer {
+  margin-top: 4.8rem;
+}
+
+.footer p {
+  font-size: 1.4rem;
+  line-height: 1.2;
+  font-weight: 400;
+}
+
+.footer img {
+  width: 100%;
+  position: relative;
+}
+
+.footer {
+  display: grid;
+  grid-template-columns: 1fr 1.5fr;
+  align-items: center;
+  justify-content: center;
+  gap: 2.4rem;
+  padding: 1.2rem;
+}
```

Comment:
- [ ] Kindly make sure your footer background in `about` page is dark as shown in the provided design. :wink: 

    - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/203545125-8c7f27b7-1ddd-41b5-bdca-12ba749d3b8e.png)

    - ### Provided design
    ![image](https://user-images.githubusercontent.com/87197412/203545076-09e892f3-d9e3-4ef9-a60c-5f5a88a956b0.png)